### ISSUE TITLE
[fix] PFE-badge variants fail a11y contrast & no darker options among the variants

### ISSUE THREAD

**Issue by Djfaucette** [patternfly/patternfly-elements#1772]
## Description of the issue

PFE-badge variants are barely visible against the background of the RHEC Search pages

### Impacted component(s)

- Filter Select

### Steps to reproduce

1. Go to [any Catalog search page](https://catalog.redhat.com/hardware/servers/search?p=1)
2. Click on any filter checkbox
3. Badge will appear next to the facet heading
4. Confirm that color contrast isn't high enough/badge outline is difficult to see

### Expected behavior

The PFE Badge icon color should be distinguishable from the background color.

### Screenshots

<img width="335" alt="Screen Shot 2021-09-15 at 1 08 57 PM" src="https://user-images.githubusercontent.com/56173867/133489421-6588a942-c19e-46a0-8dda-d83e43f3fdbb.png">

## Related issues

- #100

**Comment by bennypowers** [comment]
Closing based on https://github.com/patternfly/patternfly-elements/pull/1783#issuecomment-1049840313. If still relevant, please re-open.
